script.js

```diff
@@ -0,0 +1,31 @@
+const textsToCopyContainer = document.querySelector('#textsToTypeContainer')
```

Comment:
1. there are two slightly different names here 😅 you should make them both the same
2. whichever you go with, i think it should be `text` rather than `texts`, texts sounds like text messages or ancient texts to me

Reply:
Done